e2e/pnpm_workspace/lib/a/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@lib/a",
+    "private": true,
+    "dependencies": {
+        "@aspect-test/e": "1.0.0",
+        "@lib/b": "workspace:*"
```

Comment:
time for me to read the fucking documentation huh?

> Luckily, pnpm supports the workspace: protocol. When this protocol is used, pnpm will refuse to resolve to anything other than a local workspace package.

Reply:
🌮